src/rust/engine/serverset/src/lib.rs

```diff
@@ -390,7 +390,7 @@ mod tests {
     let s = Serverset::new(vec!["good", "bad"], backoff_config).unwrap();
     let mut runtime = tokio::runtime::Runtime::new().unwrap();
 
-    for _ in 0..2 {
+    for _ in 0..4 {
```

Comment:
Not sure what this does. Can you explain? (preferentially in the code, maybe by naming this magic number)

Reply:
Added comment